Support for mocha 10

mocha@10 was released yesterday (see https://github.com/mochajs/mocha/releases/tag/v10.0.0), are there any plans to support it?

> npm ERR! Found: mocha@10.0.0
> npm ERR! node_modules/mocha
> npm ERR!   dev mocha@"^10.0.0" from the root project
> npm ERR! 
> npm ERR! Could not resolve dependency:
> npm ERR! peer mocha@"^3.X.X || ^4.X.X || ^5.X.X || ^6.X.X || ^7.X.X || ^8.X.X || ^9.X.X" from ts-mocha@9.0.2
> npm ERR! node_modules/ts-mocha
> npm ERR!   dev ts-mocha@"^9.0.2" from the root project

--- a/package.json
+++ b/package.json
@@ -37,7 +37,7 @@
     "typescript": "3.3.3"
   },
   "peerDependencies": {
-    "mocha": "^3.X.X || ^4.X.X || ^5.X.X || ^6.X.X || ^7.X.X || ^8.X.X || ^9.X.X"
+    "mocha": "^3.X.X || ^4.X.X || ^5.X.X || ^6.X.X || ^7.X.X || ^8.X.X || ^9.X.X || ^10.X.X"
   },
   "optionalDependencies": {
     "tsconfig-paths": "^3.5.0"

